### PR TITLE
Add release notes for 6.0.0-beta1

### PR DESCRIPTION
**Pull request by suyograo**

### CODE REVIEW

File: docs/static/releasenotes.asciidoc [hunk 1]
```diff
@@ -10,12 +10,12 @@ This section summarizes the changes in the following releases:
...
 === Logstash 6.0.0-beta1 Release Notes
 
-Placeholder for beta1 release notes
-
-* Added new `logstash.yml` setting: `config.support_escapes`. When
-  enabled, Logstash will interpret escape sequences in strings in the pipeline
-  configuration.
-
+* Added new `logstash.yml` setting: `config.support_escapes`. When enabled, Logstash will interpret escape sequences in      strings in the pipeline configuration.
+* Breaking: The setting `config.reload.interval` has been changed to use time value strings such as `5m`, `10s`, etc for convenience. Previously, users had to convert this to a millisecond time value themselves.
```
Comment (jsvd) [inline docs/static/releasenotes.asciidoc]: s/millisecond/second

File: docs/static/releasenotes.asciidoc [hunk 2]
```diff
@@ -10,12 +10,12 @@ This section summarizes the changes in the following releases:
...
+* Breaking: The list of plugins bundled with Logstash beta1 release has changed. Please consult the breaking changes document for a complete list.
+* Added an x-pack feature to manage Logstash configurations centrally in Elasticsearch. We've also added a UI to manage configurations directly in Kibana without having to restart Logstash.
```
Comment (dedemorton) [inline docs/static/releasenotes.asciidoc]: Capitalize X-Pack. Can use the xpack attribute to resolve the name. Someday we'll do this for all product names, but not a priority.  

Added an {xpack} feature....

File: docs/static/releasenotes.asciidoc [hunk 3]
```diff
@@ -10,12 +10,12 @@ This section summarizes the changes in the following releases:
...
 
-Placeholder for beta1 release notes
-
-* Added new `logstash.yml` setting: `config.support_escapes`. When
-  enabled, Logstash will interpret escape sequences in strings in the pipeline
-  configuration.
-
+* Added new `logstash.yml` setting: `config.support_escapes`. When enabled, Logstash will interpret escape sequences in      strings in the pipeline configuration.
+* Breaking: The setting `config.reload.interval` has been changed to use time value strings such as `5m`, `10s`, etc for convenience. Previously, users had to convert this to a millisecond time value themselves.
+* Breaking: The list of plugins bundled with Logstash beta1 release has changed. Please consult the breaking changes document for a complete list.
```
Comment (dedemorton) [inline docs/static/releasenotes.asciidoc]: I wonder if you should say 6.0.0-beta 1 release (to be consistent with how the version will be referred to elsewhere).

File: docs/static/releasenotes.asciidoc [hunk 4]
```diff
@@ -10,12 +10,12 @@ This section summarizes the changes in the following releases:
...
+* Added an x-pack feature to manage Logstash configurations centrally in Elasticsearch. We've also added a UI to manage configurations directly in Kibana without having to restart Logstash.
+* Users can now visualize the Logstash configuration pipeline as part of the x-pack monitoring feature.
```
Comment (dedemorton) [inline docs/static/releasenotes.asciidoc]: Use {xpack} to resolve the name.

**Comment by suyograo** [comment]
@jsvd @dedemorton Addressed all your comments. Merging.

**Pull request merged**